runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -1271,6 +1271,8 @@ ClassFileOracle::walkMethodCodeAttributeAttributes(U_16 methodIndex)
 							J9CfrLocalVariableTypeTableEntry *lvttCompareEntry = &(localVariableTypeTable->localVariableTypeTable[localVariableTypeTableCompareIndex]);
 							if ((lvttEntry->startPC == lvttCompareEntry->startPC)
 								&& (lvttEntry->length == lvttCompareEntry->length)
+								&& (lvttEntry->nameIndex == lvttCompareEntry->nameIndex)
+								&& (lvttEntry->signatureIndex == lvttCompareEntry->signatureIndex) 
```

Comment:
This is a good starting point.  We should also, possibly under a separate PR, validate whether this should be checking the contents for equality, not the indexes

Reply:
Would be worth testing the RI as well

Reply:
Thats a good idea. There appears to be at least one more missing LocalVariableTypeTable check that will come after this in a separate pr.